### PR TITLE
Pnv

### PR DESCRIPTION
**Pull request by AlexandreJunod**

### CODE REVIEW

File: geocity/apps/permits/forms.py [hunk 1]
```diff
@@ -1230,10 +1230,10 @@ class PermitRequestGeoTimeForm(forms.ModelForm):
             },
             attrs={"autocomplete": "off"},
         ).start_of("event days"),
-        help_text="Cliquer sur le champ et selectionner la date planifiée de début à l'aide de l'outil mis à disposition",
+        help_text="Cliquer sur le champ et selectionner la date de début à l'aide de l'outil mis à disposition",
```
Comment (rbovard) [inline geocity/apps/permits/forms.py:1233]:
```suggestion
        help_text="Cliquer sur le champ et sélectionner la date de début à l'aide de l'outil mis à disposition",
```

File: geocity/apps/permits/forms.py [hunk 2]
```diff
@@ -1243,7 +1243,7 @@ class PermitRequestGeoTimeForm(forms.ModelForm):
             },
             attrs={"autocomplete": "off"},
         ).end_of("event days"),
-        help_text="Cliquer sur le champ et selectionner la date planifiée de fin à l'aide de l'outil mis à disposition",
+        help_text="Cliquer sur le champ et selectionner la date de fin à l'aide de l'outil mis à disposition",
```
Comment (rbovard) [inline geocity/apps/permits/forms.py:1246]:
```suggestion
        help_text="Cliquer sur le champ et sélectionner la date de fin à l'aide de l'outil mis à disposition",
```

File: geocity/apps/permits/models.py [hunk 3]
```diff
@@ -131,6 +131,19 @@ class StepType(enum.Enum):
 StepType.do_not_call_in_templates = True
 
 
+Group.add_to_class(
+    "shortname",
+    models.CharField(
+        _("nom court"),
+        max_length=32,
+        help_text=_(
+            "Nom affiché par defaut dans les différentes étapes du formulaire. Ne s'affichage pas dans l'admin. (max. 32 caractères)"
```
Comment (rbovard) [inline geocity/apps/permits/models.py:140]:
```suggestion
            "Nom affiché par défaut dans les différentes étapes du formulaire, ne s'affiche pas dans l'admin (max. 32 caractères)"
```

File: geocity/apps/permits/models.py [hunk 4]
```diff
@@ -1142,6 +1155,14 @@ class WorksObjectType(models.Model):
     permanent_publication_enabled = models.BooleanField(
         _("Autoriser la mise en consultation sur une durée indéfinie"), default=False
     )
+    shortname = models.CharField(
+        _("nom court"),
+        max_length=32,
+        help_text=_(
+            "Nom affiché par defaut dans les différentes étapes du formulaire. Ne s'affichage pas dans l'admin. (max. 32 caractères)"
```
Comment (rbovard) [inline geocity/apps/permits/models.py:1162]:
```suggestion
            "Nom affiché par défaut dans les différentes étapes du formulaire, ne s'affiche pas dans l'admin (max. 32 caractères)"
```

**Review by rbovard — COMMENTED** [review]
Some typo ;)

I guess the `print`s are reflecting the WIP status

**Review by monodo — APPROVED** [review]

**Pull request merged**